ovs_performance.py

```diff
@@ -3072,12 +3081,13 @@ def main():
     else:
         raise ValueError("No support for this protocol!!")
 
-    with open(csv_file, 'wb') as csvfile:
+    with open(csv_file, 'w') as csvfile:
```

Comment:
Any specific reason for changing this? guess window machines now add cr/lf.

Reply:
The unicode error ("wb' was writing bytes to the object whereas 'w' writes as string)